client/modules/core/main.js

```diff
@@ -243,21 +221,17 @@ export default {
       return false;
     }
 
-    //
-    // actual logic block to check permissions
-    // we'll bypass unecessary checks during
-    // a user logging, as we'll check again
-    // when everything is ready
-    //
+    /*
+      Checks for permissions. We'll skip performing some checks if the user is
```

Comment:
I think we can leave this part of the comment as is. It seems valid and clear (IMO)

Reply:
Ok. I'll restore the former comment.